app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -306,8 +306,11 @@ function handleFocus(event) {
     area.blur();
   } else {
     var form = event.target.closest(".comment-form");
-    form.classList.add("comment-form--initiated");
+    if (form) {
+      form.classList.add("comment-form--initiated");
+    }
```

Comment:
This change is because the notification comment box is not part of the 'comment-form', but can make use of the same focus handler